R/MsBackendTimsTof-functions.R

```diff
@@ -0,0 +1,15 @@
+.header <- function(x, file = character()) {
+  tms <- OpenTIMS(file)
+  x@frames <- tms@frames
+  x@indexes <- cbind(
```

Comment:
Minor point to figure out at the beginning: isn't `indices` the correct/better term?
If the slot is named `index` (or its plural form) isn't `frame_index`, `scan_index` and `spectrum_index` redundant. I would favor `frame`, `scan` and `spectrum` as column names instead.

Reply:
Good point @sgibb . I was proposing the `"_index"` names to make it clear that these are integer indices and nothing else - but since the slot is already named accordingly I agree, we can drop that.

Also, along the line, maybe better to rename the function `.header` function to `.initialize` @andreavicini or something like that. For `mzR` it made sense to call that function `.header`, but here maybe not so mucht.